contractcourt/channel_arbitrator.go

```diff
@@ -1865,8 +1865,8 @@ func (c *ChannelArbitrator) checkRemoteDanglingActions(
 			continue
 		}
 
-		log.Infof("ChannelArbitrator(%v): immediately failing "+
-			"htlc=%x from remote commitment",
+		log.Infof("ChannelArbitrator(%v): fail dangling htlc=%x from "+
+			"local/remote commitments diff",
```

Comment:
Why "local/remote" and not just remote? From the surrounding comments I would expect only "remote", but could be wrong due to the lack of familiarity with this part of the code base.